.github/workflows/update.yml

```diff
@@ -27,22 +27,26 @@ jobs:
           ref: ${{ steps.config.outputs.ref }}
           submodules: recursive
 
-      - name: Install Nix
-        uses: cachix/install-nix-action@v14.1
+      - name: 'Install Nix'
+        uses: cachix/install-nix-action@v15
         with:
           extra_nix_config: |
-            substituters = http://cache.nixos.org https://hydra.iohk.io
+            substituters = http://cache.nixos.org https://cache.iog.io
             trusted-public-keys = cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY= hydra.iohk.io:f/Ea+s+dFdN+3Y/G+FDgSq+a5NEWhJGzdjvKNGv0/EQ=
-          install_url: "https://releases.nixos.org/nix/nix-2.3.16/install"
 
-      - name: Install Cachix
+      - name: 'Install Cachix'
         uses: cachix/cachix-action@v10
         with:
           name: runtimeverification
+          extraPullNames: kore
           signingKey: '${{ secrets.CACHIX_SIGNING_KEY }}'
 
+
       - name: Materialize
-        run: ./nix/rematerialize.sh
+        run: GC_DONT_GC=1 nix run .#update-cabal
```

Comment:
Should we delete the old script then?
But maybe it is useful to keep it so one can run the update locally. Then we would want to have the `nix run ...` commands in that script.

Reply:
No, the old script works with nix <2.4 without flakes. It should be removed if/when we get rid of non flake nix stuff in the repo

Reply:
:man_shrugging: OK then we keep it as it is for now.